lib/ClangImporter/DWARFImporter.cpp

```diff
@@ -21,11 +21,14 @@ void DWARFImporterDelegate::anchor() {}
 /// loading of types is done on demand, this class is effectively empty.
 class DWARFModuleUnit final : public LoadedFile {
   ~DWARFModuleUnit() = default;
-  ClangImporter::Implementation &owner;
+  ClangImporter::Implementation &Owner;
+  std::string ModuleName;
```

Comment:
You don't need to store this; it'll be `getParentModule()->getIdentifier()`. (I suggest using Identifier instead of StringRef throughout.)